Frontend/src/data/rental/inputs.js

```diff
@@ -33,11 +35,32 @@ const updateToggleStatus = (context, itemExistsMoreThanOnce) => {
 };
 
 function getRecentEmployees() {
-  var employeeList = {};
+  var employeeObj = {};
   for (let employee of get(recentEmployeesStore)) {
-    employeeList[employee] = employee;
+    employeeObj[employee] = employee;
   }
-  return employeeList;
+  return employeeObj;
+}
+
+function suggestReceivingEmployee(context) {
+  if (context.doc.receiving_employee != "") {
+    return context.doc.receiving_employee;
+  }
+  var employeeObj = getRecentEmployees();
+
+  // Object.keys() is failing the cypress tests :-/ so need
+  // to manually create the array like a stone-age person
+  var keys = new Array();
+  for (var key in employeeObj) {
+    keys[keys.length] = key;
+  }
+
+  var mostRecent = keys[keys.length - 1];
```

Comment:
- Why are you converting recent employes into an object an then into an array? Couldn't you just take the last element out of `get(recentEmployeesStore)` here?
- I think recentEmployeesStore is currently always empty because we never write something in this store?

Reply:
The function is used elsewhere, which needs the format `{'displayvalue': value}` (quickset of employees). 

You are right, `get(recentEmployeesStore)` is much easier ^^ didn't think of it

It's not empty, we write to it in line `111` of `rental/onSave.js`
https://github.com/leih-lokal/LeihLokalVerwaltung/blob/a4de80c77d01abe9824a6029a81dd03cfdbea4f0/Frontend/src/data/rental/onSave.js#L93